doc/article/en-US/angular1-to-aurelia.md

```diff
@@ -0,0 +1,526 @@
+			 * for HTTP requests, or client-side routing templates - but after Angular has bootstrapped,
+			 * configuration is locked.
+			 */
+			...
+		}]);
+	</source-code>
+</code-listing>
+
+Aurelia uses a similar configuration model, but services don't have to be configured before they are used; configuration is generally only used for global resource locations and framework plugins.
+
+<code-listing heading="Aurelia Configuration Example">
+	<source-code lang="ES 2015/2016">
+		export function configure(aurelia) {
+		  aurelia.use
+		    .standardConfiguration() //
+		    .plugin('aurelia-animator-css')
+		    .feature('someFeature')
+		    .globalResources('resources/my-element', 'resources/my-other-element');
+		
+		  aurelia.start().then(a => a.setRoot());
+		}
+	</source-code>
+</code-listing>
+
+This means that certain important service configurations (especially routing and HTTP) can be configured during application runtime. We'll touch upon this later.
+
+### Using Modules and Dependency Injection
+
+Third step - we'll migrate our modules from Angular to Aurelia. Like Aurelia, Angular supported modules and dependency injection; however, Angular used a non-standard, Angular-specific module format. Whether loaded through a loader or through generic script tags, Angular declared a global `angular` object. Using any controller, service, or other functionality with Angular meant that it had to be registered with an Angular module at bootstrap time in order to be used. An example Angular configuration might look like the code below.
+
+<code-listing heading="Angular Component Definitions">
+	<source-code lang="ES 2015/2016">
+		angular.module('myModule')
+			.service('myService', ['$q', '$http',function ($q, $http) {
+				...
+			}]);
+		
+		angular.module('myModule')
+			.service('anotherService', ['$q', '$router',function ($q, $router) {
+				...
+			}]);
+		
+		angular.module('myModule')
+			.factory('myFactory, ['$storage', function ($storage) {
+		
+			}]);
+		
+		angular.module('myModule')
+			.value('myValue', "value")
+			... // and so on
+	</source-code>
+</code-listing>
+
+Aurelia handles things a little differently. Instead of framework-specific modules, Aurelia leverages native Javascript modules using either the ES6 import statement or the `require`-style AMD syntax. (These end up transpiled to AMD modules for browser use.) When registering objects with Angular modules Angular made a distinction between `service`, `factory`,  `constant`, and `value` - all syntactic sugar for `provider`. Aurelia, in contrast, makes no special framework distinction between "services", "factories", and other module types - any viewmodels or services are just simple Javascript modules, simplifying the way you can write your code.
+
+So, the angular code above might end up looking more like this:
+
+<code-listing heading="Aurelia Component Definitions">
+	<source-code lang="ES 2015/2016">@inject(Q, HttpClient)
+		export class myService {
```

Comment:
You haven't added imports for dependencies not defined in this file - why, is it conscious decision?